dockstore-webservice/src/main/java/io/dockstore/webservice/resources/WorkflowResource.java

```diff
@@ -2092,4 +2148,38 @@ public Response handleGitHubBranchDeletion(@ApiParam(hidden = true) @Parameter(h
         githubWebhookDelete(repository, gitReference, username);
         return Response.status(HttpStatus.SC_NO_CONTENT).build();
     }
+
+    @GET
+    @Path("/{workflowId}/workflowVersions/{workflowVersionId}/orcidAuthors")
+    @UnitOfWork(readOnly = true)
+    @ApiOperation(value = "Retrieve ORCID author information for a workflow version",  hidden = true)
```

Comment:
Is there a reason this api operation should be hidden?

Reply:
I was following the same pattern as some of the newer endpoints like `getWorkflowVersionsSourcefiles` and `postParsedInformation` which hide the endpoints from swagger. I was under the impression that we should use openapi whenever possible?